spec/decorators/ncr_work_order_decorator_spec.rb

```diff
@@ -18,4 +18,11 @@
     wo = create(:ncr_work_order, :with_approvers).decorate
     expect(wo.current_approver_email_address).to eq(wo.approvers.first.email_address)
   end
+
+  it "returns approver email address based on proposal status" do
```

Comment:
don't want to be a nitpicker here, but it might be easier to understand the behavior with a separate test for each use case.

`it "returns current approver email address for pending proposals" do`

`it "returns final approver email address for approved proposals" do`